webhook_server/utils/github_retry.py

```
@@ -0,0 +1,128 @@
+        return False
+
+    if isinstance(ex, (RequestsConnectionError, MaxRetryError, ResponseError)):
+        return True
+
+    error_str = str(ex)
+    return any(substring in error_str for substring in _RETRYABLE_SUBSTRINGS)
+
+
+async def github_api_call[T](
+    func: Callable[..., T],
+    *args: Any,
```

Ruff (0.15.9)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `*args`

(ANN401)